solutii/cristi.danila/Xand0/remiza.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
Nu trebuie sa adaugi un fisier in acest director. Trebuie respectata structura prezentat la laborator.
Structura corecta `web/solutii/<nume_prenume>/*` nu `solutii/<nume_prenume>/*`.
Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).